sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosBridgeInternal.java

```diff
@@ -130,6 +131,12 @@ public static CosmosClientBuilder cloneCosmosClientBuilder(CosmosClientBuilder b
         return copy;
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static CosmosClientBuilder setUsingState(CosmosClientBuilder cosmosClientBuilder, CosmosClientState state) {
```

Comment:
Will intel sense show these contracts?

Also can it be made deprecated to start with?

Reply:
This is an established pattern in the SDK. There are many other methods in this file with the same pattern. We should follow the same pattern here.

Reply:
removed the CosmosBridgeInternal completely.
see this commit please:
https://github.com/Azure/azure-sdk-for-java/commit/2f31869e9c5d84b230f1664070455ef4a20f81f2

This is a pattern to get rid of BridgeInternal classes.